Gemfile

```diff
@@ -107,7 +107,7 @@ gem 'httparty', '~> 0.13'
 gem 'httmultiparty', '~> 0.3.16'
 gem 'jquery-rails', '~> 4.2.1'
 gem 'huginn_agent', '~> 0.4.0'
-gem 'json', '~> 1.8.1'
+gem 'json'
```

Comment:
I had this changed to `gem 'json', '~> 1.8.5'` in https://github.com/cantino/huginn/issues/1874, but if this works, even better!